src/parsing/css-style-value-parsing.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  var parsers = {};
+  parsers[CSSNumberValue] = internal.parsing.consumeNumber;
+  parsers[CSSLengthValue] = internal.parsing.consumeLength;
+  parsers[CSSTransformValue] = internal.parsing.consumeTransform;
+
+  function consume(consumer, string) {
```

Comment:
> This is a helper function to deal with the fact that there may not be an entry in the parsers map above
Since it's only called once I might remove it.

> There's a bug in this function anyway, it's not passing in the property.
Why would it need to pass in the property?